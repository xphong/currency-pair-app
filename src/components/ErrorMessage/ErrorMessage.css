.error {
  color: red;
  margin: 12px 0;
}
