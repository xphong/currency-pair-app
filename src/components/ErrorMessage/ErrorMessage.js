import React from 'react';

import styles from './ErrorMessage.css';

const ErrorMessage = props => {
  return (
    <div className={styles.error}>
      {props.children}
    </div>
  );
};

export default ErrorMessage;
