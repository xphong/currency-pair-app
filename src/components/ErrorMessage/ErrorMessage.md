### Default Style

```js
<ErrorMessage>Error</ErrorMessage>
```
