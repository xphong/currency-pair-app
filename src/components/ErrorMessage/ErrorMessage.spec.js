import React from 'react';

import ErrorMessage from './ErrorMessage';

describe('ErrorMessage Component', () => {
  it('should render ErrorMessage', () => {
    const wrapper = shallow(<ErrorMessage>error</ErrorMessage>);

    expect(wrapper).toMatchSnapshot();
  });
});
