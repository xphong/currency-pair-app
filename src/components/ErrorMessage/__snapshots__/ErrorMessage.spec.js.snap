// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage Component should render ErrorMessage 1`] = `
<div
  className="error"
>
  error
</div>
`;
